manifests/base/runtimes/kustomization.yaml

```diff
@@ -1,8 +1,4 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - deepspeed_distributed.yaml
-  - mlx_distributed.yaml
-  - mpi_distributed.yaml
   - torch_distributed.yaml
```

Comment:
I don't think we want to use any of the upstream runtimes. For PyTorch we'll use our own runtime relying on our training images.